opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -218,8 +220,6 @@ class SimulatorFullyImplicitBlackoilEbos
             ebosSimulator_.setEpisodeIndex(-1);
             ebosSimulator_.setEpisodeLength(0.0);
             ebosSimulator_.setTimeStepSize(0.0);
-            // Make cache up to date. No need for updating it in elementCtx.
-            ebosSimulator_.model().invalidateAndUpdateIntensiveQuantities(/*timeIdx=*/0);
```

Comment:
it's probably okay, but this in separation seems like we change from doing it every iteration to just once up front ?

Reply:
Not every iteration,  only when timer.initialStep() is true. And that will never be true when we restart.